packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/TransformHelper.java

```diff
@@ -107,4 +109,87 @@ public static void processTransform(ReadableArray transforms, double[] result) {
       MatrixMathHelper.multiplyInto(result, result, helperMatrix);
     }
   }
+
+
+  private static double convertToDegrees(ReadableMap transformMap, String key) {
+    double value;
+    boolean inRadians = true;
+    if (transformMap.getType(key) == ReadableType.String) {
+      String stringValue = transformMap.getString(key);
+      if (stringValue.endsWith("deg")) {
+        inRadians = false;
+      }
+      if (stringValue.endsWith("rad") || stringValue.endsWith("deg")) {
+        stringValue = stringValue.substring(0, stringValue.length() - 3);
+      }
+      value = Float.parseFloat(stringValue);
+    } else {
+      value = transformMap.getDouble(key);
+    }
+    return inRadians ? MatrixMathHelper.radiansToDegrees(value) : value;
+  }
+
+
+  public static boolean tryProcessTransformBySkiaMatrix(ReadableArray transforms, Matrix matrix,
+                                                        int viewWidth, int viewHeight) {
+    assert matrix != null;
+    matrix.reset();
```

Comment:
Since we're allocating a new Matrix in the caller anyway, could we just return a Matrix here if successful (and null otherwise?)